src/plugins/visualizations/public/legacy/build_pipeline.ts

```diff
@@ -309,26 +302,6 @@ export const buildPipelineVisFunction: BuildPipelineVisFunction = {
 };
 
 const buildVisConfig: BuildVisConfigFunction = {
-  table: (schemas, visParams = {}) => {
```

Comment:
Given that you have moved to using `toExpressionAST` this can be removed, but a user can also use the raw expression syntax to create a table. Can the new expression function handle the saem props as the old one. Cause if not, that would be a breaking change

Reply:
should we add it back for now? then create an issue and remove it in 3.0.0?

Reply:
I tested with the following example 

```
old expression:

"opensearchDashboards | opensearch_dashboards_context | opensearchaggs index='ff959d40-b880-11e8-a6d9-e546fe2bba5f' metricsAtAllLevels=false partialRows=false
 aggConfigs='[{"id":"1","enabled":true,"type":"count","params":{},"schema":"metric"}]'  
| opensearch_dashboards_table 
  visConfig='{"title":"","perPage":10,"showPartialRows":false,"showMetricsAtAllLevels":false,"sort":{"columnIndex":null,"direction":null},"showTotal":false,"totalFunc":"sum","percentageCol":"","dimensions":{"metrics":[{"accessor":0,"format":{"id":"number"},"params":{},"label":"Count","aggType":"count"}],"buckets":[]}}'"


new expression:

"opensearchDashboards | opensearch_dashboards_context | opensearchaggs index="ff959d40-b880-11e8-a6d9-e546fe2bba5f" metricsAtAllLevels=false partialRows=false 
  aggConfigs="[{\"id\":\"1\",\"enabled\":true,\"type\":\"count\",\"params\":{},\"schema\":\"metric\"}]" includeFormatHints=false
| opensearch_dashboards_table 
  visConfig="{\"perPage\":10,\"percentageCol\":\"\",\"showPartialRows\":false,\"showMetricsAtAllLevels\":false,\"showTotal\":false,\"totalFunc\":\"sum\",\"metrics\":[{\"accessor\":0,\"format\":{\"id\":\"number\"},\"params\":{},\"label\":\"Count\",\"aggType\":\"count\"}],\"buckets\":[]}""
```

You could see some diff, like `sort` and `title`. `title` can be added later. it is used for saving the vis. `sort` might be diff because we decide to use OUI datagrid. 

I hard coded the old expression into loader.ts to make the new vis using the older expression:

<img width="1643" alt="Screen Shot 2022-09-15 at 16 08 36" src="https://user-images.githubusercontent.com/79961084/190524066-a669f6a5-cb57-40b8-87a5-bdc7989c6e79.png">

so far I don't see error and this is because I haven't restore sort back. I think for the very early stage, it is hard to say whether we can guaranteed bwc but I would like to proceed. If previous usage is affected, for example saved table vis is not loaded, then we need to target the new table vis for 3.0. There are ways we could work around, like saved object migration. I will create a sub issue to track this.